docs/MIGRATING_TO_1_0.md

```diff
@@ -0,0 +1,163 @@
+# Migrating from 0.3 to 1.0
+
+## Split in two modules
+
+The core of immutad●t has been rewritten without [lodash](https://lodash.com/), and immutad●t now has two modules [immutadot](https://www.npmjs.com/package/immutadot) and [immutadot-lodash](https://www.npmjs.com/package/immutadot-lodash).
+
+### immutadot module
+
+immutadot module now contains exclusively functions based on [ES2015+ standard library](https://mdn.io/JavaScript/Reference).
+
+For a complete list see [immutadot's API documentation](https://zenika.github.io/immutadot/immutadot/1.0/).
+
+immutad●t's organization in namespaces has changed a little, see [Namespaces modifications](#namespaces-modifications) for more details.
+
+Some functions such as [`filter()`](https://zenika.github.io/immutadot/immutadot/1.0/array.html#.filter) and `map(https://zenika.github.io/immutadot/immutadot/1.0/array.html#.map)` are now available in immutadot and in immutadot-lodash. This is because immutadot's version handles only arrays whereas immutadot-lodash's version handles collections (which includes objects), it is up to you to choose which version suits best your needs.
+
+### immutadot-lodash module
+
+All immutad●t functions directly based on lodash utilities (such as [`mapValues()`](https://zenika.github.io/immutadot/immutadot-lodash/1.0/object.html#.mapValues)) have been moved into immutadot-lodash.
+
+For a complete list see [immutadot-lodash's API documentation](https://zenika.github.io/immutadot/immutadot-lodash/1.0/).
+
+Such cases are easily migrated:
+```diff
+-import { mapValues } from 'immutadot'
++import { mapValues } from 'immutadot-lodash'
+
+ mapValues(obj, 'nested.prop', val => val + 1)
+```
+
+All immutadot-lodash functions have kept their former namespace, which is their original lodash namespace.
+
+## Curried `obj` parameter
+
+immutad●t functions now support currying the `obj` (leftmost) parameter:
+
+```js
+set('nested.prop', 'val')(obj)
+```
+
+This allows them to be used in [`flow()`](https://zenika.github.io/immutadot/immutadot/1.0/core.html#.flow) (see [Chained operations](#chained-operations)).
+
+Partial functions returned by calls without `obj` are unary, therefore you may call them with more than one parameter, only the first one will be used:
+
+```js
+set('nested.prop', 'val')(obj, discarded1, discarded2)
+```
+
+## Chained operations
+
+[`chain()`](https://zenika.github.io/immutadot/immutadot/0.3/seq.html#.chain) has been removed in favor of [`flow()`](https://zenika.github.io/immutadot/immutadot/1.0/core.html#.flow).
+
+The main reasons behind this are that `chain` had some major drawbacks:
+ - Imports the whole of immutad●t
```

Comment:
Imports all immutad●t ?

Reply:
I find "all immutad●t" less clear

Reply:
Middleground: all of immutad●t's functions ?